src/external/visualization_integration.cuh

```diff
@@ -38,7 +39,7 @@ __host__ std::string vector_double_to_json_array(const std::vector<double> &v);
  * JSON is generated: `{"x": [0.000000, 0.500000], "y": [0.000000, 1.000000], "z": [0.000000, 2.000000]}`. Then it is
  * sent to the given url as an HTTP POST request with `Content-Type: application/json`
  *
- * @param url HTTP url to send data to (protocols different from http are not supported)
+ * @param urls HTTP two urls each to send data to. One for particles array and one for polyhedron. (protocols different from http are not supported)
```

Comment:
```suggestion
 * @param urls HTTP two urls to send data to. First for a particle array, second for a polyhedron (protocols different
 * from http are not supported)
```

It would also be good to replace "one URL for this and one for that" with "first URL for this and second/another for that" in other places in the code/readme